src/database/migrations/0012-create-platform_provider_responses.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE platform_provider_responses (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  external_id VARCHAR NOT NULL,
+  platform_provider VARCHAR NOT NULL,
+  data JSON NOT NULL,
```

Comment:
In general, prefer the `JSONB` data type over the `JSON` data type:

```suggestion
  data JSONB NOT NULL,
```